packages/docs/docs/getting-started.md

```diff
@@ -183,3 +183,34 @@ $ grep -i 'starting modelcall' packages/ai-jsx/llmx.log | yarn workspace ai-jsx
   }
 ],
 ```
+
+## Beyond Text: Image Generation
+
+You are not restricted by text models. Let's create an image for the story as well:
+
+```tsx title="index.tsx"
+function StoryWithImage() {
+  const story = memo(<WriteStory />);
+  return (
+    <>
+      Banner URL: <ImageGen clipLongPrompt>Generate an image for this story: {story}</ImageGen>
```

Comment:
I would use the `// highlight-next-line` comment here to emphasize that this is the most important line in the code snippet.

Reply:
Good to know, thanks.